WebRoot/public/PasswordRecovery.jsp

```diff
@@ -98,6 +101,32 @@
 			accountInput : "<%=accountInput%>"
 		};
 		new ZmPasswordRecoveryDialog(params).popup();
+		var changeTag = document.getElementsByClassName("DwtDialogTitle")[0];
+		changeTag.outerHTML = "";
+		changeTag = document.getElementsByClassName("horizSep")[0];
+		changeTag.outerHTML = "";
+		changeTag = document.querySelectorAll('[align=right]');
+		changeTag[0].align="left";
+		changeTag = document.querySelectorAll('table[role="presentation"][width="100%"]');
+		changeTag[0].removeAttribute("width");
```

Comment:
We are doing same thing repeatedly, can't we have a helper function for the same?

Reply:
will try to refactor this code in upcoming tickets https://jira.corp.synacor.com/browse/PREAPPS-4328